src/handbook/peopleops/hiring/reference-check.md

```diff
@@ -0,0 +1,54 @@
+---
+navTitle: Reference Check
+---
+
+Reference checks are a much needed final stage in the hiring process, conducted
+after the interview stages but before extending an offer. These conversations
+provide valuable insights into a candidate's past performance, work style, and
+potential fit with FlowFuse.
+
+## When to Conduct Reference Checks
+
+Reference checks should be conducted after all interviews are complete and the
+hiring team has identified a preferred candidate. We request 3 professional
+references from the candidate, preferably including:
+- A former direct manager
+- A peer or colleague
+- A direct report (if applicable for the role)
+
+FlowFuse will conduct 1 or 2 reference check calls, typically prioritizing
+conversations with former managers who can speak to the candidate's performance.
+
+## Structure of a Reference Check Call
+
+The reference check is typically a 20-30 minute conversation conducted over the phone or video call. The structure should be:
+
+1. Introduce yourself and FlowFuse, and explain the purpose of the call.
+2. Confirm the reference's relationship to the candidate and time worked together.
+3. Ask the [reference check questions](#reference-check-questions).
+4. Thank the reference for their time and insights.
+
+## Reference Check Questions
+
+During the reference check, aim to understand:
```

Comment:
```suggestion
These questions are intended to surface working style, strengths, and growth areas to support onboarding and early management. When speaking with peers or colleagues, ask for concrete examples and specific situations rather than general impressions.

During the reference check, aim to understand:
```